app/subscriber/src/components/section/styled/PageSection.tsx

```diff
@@ -7,7 +7,7 @@ export const PageSection = styled.div<{ $ignoreMinWidth?: boolean; $ignoreLastCh
   background: ${(props) => props.theme.css.bkSecondary};
   box-shadow: ${(props) => props.theme.css.boxShadow};
   align-self: stretch;
-  margin: 1rem;
+  margin: 0 1rem 1rem 1rem;
```

Comment:
@kylgarmor - seems like this is ok to do and let's us remove the other margin 0s for the top. but let me know if you think otherwise!

Reply:
perfect!